.rubocop.yml

```diff
@@ -99,25 +101,32 @@ Style/ZeroLengthPredicate:
 Rails:
   Enabled: true
 
+Rails/ApplicationJob:
+  Enabled: false
+
+Rails/ApplicationRecord:
+  Enabled: false
+
+Rails/DynamicFindBy:
+  Enabled: false
+
+Rails/FilePath:
+  Exclude:
+    - 'spec/abilities/**/*'
+
 Rails/OutputSafety:
   Exclude:
-    - 'spec/views/hyrax/my/works/_list_works.html.erb_spec.rb'
-    - 'app/helpers/hyrax/hyrax_helper_behavior.rb'
-    - 'app/helpers/hyrax/collections_helper.rb'
-    - 'app/helpers/hyrax/content_block_helper_behavior.rb'
     - 'app/builders/hyrax/form_builder.rb'
     - 'app/helpers/hyrax/citations_behaviors/formatters/apa_formatter.rb'
     - 'app/helpers/hyrax/citations_behaviors/formatters/chicago_formatter.rb'
     - 'app/helpers/hyrax/citations_behaviors/formatters/mla_formatter.rb'
+    - 'app/helpers/hyrax/collections_helper.rb'
+    - 'app/helpers/hyrax/content_block_helper_behavior.rb'
+    - 'app/helpers/hyrax/hyrax_helper_behavior.rb'
```

Comment:
It was an issue; I just fixed the sorting of those lines.  Sorry for the confusion.